distribution/engine/THIRD-PARTY/NOTICE

```diff
@@ -431,26 +416,11 @@ The license file can be found at `licenses/MIT`.
 Copyright notices related to this dependency can be found in the directory `org.graalvm.js.js-language-24.0.0`.
 
 
-'llvm-api', licensed under the New BSD License (3-clause BSD license), is distributed with the engine.
-The license file can be found at `licenses/BSD-3-Clause`.
-Copyright notices related to this dependency can be found in the directory `org.graalvm.llvm.llvm-api-24.0.0`.
-
-
 'polyglot', licensed under the Universal Permissive License, Version 1.0, is distributed with the engine.
 The license file can be found at `licenses/Universal_Permissive_License__Version_1.0`.
 Copyright notices related to this dependency can be found in the directory `org.graalvm.polyglot.polyglot-24.0.0`.
 
 
-'python-language', licensed under the MIT License, is distributed with the engine.
-The license file can be found at `licenses/MIT`.
-Copyright notices related to this dependency can be found in the directory `org.graalvm.python.python-language-24.0.0`.
-
-
-'python-resources', licensed under the MIT License, is distributed with the engine.
-The license file can be found at `licenses/MIT`.
-Copyright notices related to this dependency can be found in the directory `org.graalvm.python.python-resources-24.0.0`.
```

Comment:
Are we no longer distributing the Python language with Enso? I thought that with `--jvm` option it is still somehow picked up, but how is it picked up if it's not present in the distribution?

Reply:
(I probably don't understand something, so I'd appreciate an explanation to be able to understand what has changed in terms of Python availability/distribution with our artifacts)

Reply:
No, we do distribute `python-resources` and co. Just the legal tool isn't able to figure it up. Who knows where exactly it tried to find out the JARs to analyze?

Reply:
All sbt projects that are distributed are listed in `GatherLicenses.distributions := ` setting.

For engine, that is:
```
    Distribution.sbtProjects(
      runtime,
      `engine-runner`,
      `language-server`
    )
```

Should we add something more?